client/app/cda/details/home/detail/edit/cda-detail-edit.html

```diff
@@ -1,63 +1,40 @@
 
-            <label for="crushTunable" data-translate="cda_detail_edit_crushTunable" class="oui-label"></label>
-            <div class="oui-select">
-                <select class="oui-select__input"
-                    id="crushTunable" name="crushTunable"
-                    data-ng-model="CdaDetailEditCtrl.model.crushTunable"
-                    data-ng-options="crushTunableValue | lowercase for crushTunableValue in CdaDetailEditCtrl.crushTunableValues"
-                    required />
-                <i class="oui-icon oui-icon-chevron-down"></i>
-            </div>
-            <div class="text-danger" data-ng-messages="editDetailForm.crushTunable.$error" data-ng-if="editDetailForm.crushTunable.$dirty">
-                <p class="animate-fadeInLeft-enter animate-fadeOutRight-leave" >
-                    <span data-ng-message="required" data-translate="cda_detail_edit_crushTunable_required"></span>
-                </p>
-            </div>
+        <h2 class="oui-modal__title" ng-bind="::'cda_detail_edit_title' | translate "></h2>
+        <div class="text-center">
+            <oui-spinner data-ng-if="CdaDetailEditCtrl.saving"></oui-spinner>
         </div>
-    </div>
 
-    <div class="oui-modal__footer">
-        <button type="reset"
-                class="oui-button oui-button_secondary"
-                title="{{ ::'common_cancel' | translate }}"
-                data-ng-click="CdaDetailEditCtrl.closeModal()"
-                data-ng-disabled="CdaDetailEditCtrl.saving"
-                form="editDetailForm">
-            <span data-translate="common_cancel"></span>
-        </button>
-        <button type="button"
-                class="oui-button oui-button_primary"
-                title="{{ ::'common_modify' | translate }}"
-                data-ng-click="editDetailForm.$valid && CdaDetailEditCtrl.editCluster()"
-                data-ng-disabled="CdaDetailEditCtrl.saving || (editDetailForm.$error && !editDetailForm.$valid)"
-                form="editDetailForm">
-            <span data-translate="common_modify"></span>
-        </button>
-    </div>
+        <div data-ng-if="!CdaDetailEditCtrl.saving">
+            <oui-field label="{{ ::'cda_detail_edit_label' | translate }}">
+                <input class="oui-input"
+                       type="text"
+                       id="label"
+                       name="label"
+                       data-ng-model="CdaDetailEditCtrl.model.label"
+                       data-ng-maxLength="CdaDetailEditCtrl.options.label.maxLength"
+                       required>
+            </oui-field>
+
+            <oui-field label="{{ ::'cda_detail_edit_crushTunable' | translate }}">
+                <oui-select
+                    name="crushTunable"
+                    title="{{ ::'cda_detail_edit_crushTunable' | translate }}"
+                    placeholder="{{ ::'cda_detail_edit_crushTunable' | translate }}"
+                    model="CdaDetailEditCtrl.model.crushTunable"
```

Comment:
data-ng-model

Reply:
mmh not ng-modal, but data-model !

Reply:
model is ok

Reply:
We don't use `data-` prefix anymore on the managers?

Reply:
I see.. yes we should add data-